onfido-java/pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
-            <version>1.9</version>
+            <version>1.13</version>
```

Comment:
Note: Updating this to resolve a Snyk error pointed out by @formerllamafarmer (Ryan)